packages/camera/camera/example/lib/main.dart

```diff
@@ -196,14 +198,14 @@ class _CameraExampleHomeState extends State<CameraExampleHome>
 
   Future<void> _handleScaleUpdate(ScaleUpdateDetails details) async {
     // When there are not exactly two fingers on screen don't scale
-    if (_pointers != 2) {
+    if (controller == null || _pointers != 2) {
       return;
     }
 
     _currentScale = (_baseScale * details.scale)
-        .clamp(_minAvailableZoom, _maxAvailableZoom);
+        .clamp(_minAvailableZoom ?? 1, _maxAvailableZoom ?? 1);
```

Comment:
If we always need values, then lets make them non-nullable and initialized to 1 (and if the thing they are set to later is nullable, add a `?? 1` at that point).